<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Widget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_r" native="true">
       <property name="styleSheet">
        <string notr="true">background: qlineargradient(
        spread:pad,
        x1:0, y1:0,
        x2:1, y2:0,
        stop:0 rgba(0, 0, 0, 0),
        stop:1 rgba(255, 0, 0, 255));

</string>
       </property>
       <widget class="QPushButton" name="RedButton_up">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/button_up.png</normaloff>:/button_up.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="RedButton_down">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/button_down.png</normaloff>:/button_down.png</iconset>
        </property>
       </widget>
       <widget class="QLabel" name="brightness_r">
        <property name="geometry">
         <rect>
          <x>35</x>
          <y>120</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_g" native="true">
       <property name="styleSheet">
        <string notr="true">background: qlineargradient(
        spread:pad,
        x1:0, y1:0,
        x2:1, y2:0,
        stop:0 rgba(0, 0, 0, 0),
        stop:1 rgba(0, 255, 0, 255));</string>
       </property>
       <widget class="QPushButton" name="GreenButton_up">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/button_up.png</normaloff>:/button_up.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="GreenButton_down">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/button_down.png</normaloff>:/button_down.png</iconset>
        </property>
       </widget>
       <widget class="QLabel" name="brightness_g">
        <property name="geometry">
         <rect>
          <x>35</x>
          <y>120</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_b" native="true">
       <property name="styleSheet">
        <string notr="true">background: qlineargradient(
        spread:pad,
        x1:0, y1:0,
        x2:1, y2:0,
        stop:0 rgba(0, 0, 0, 0),
        stop:1 rgba(0, 0, 255, 255));</string>
       </property>
       <widget class="QPushButton" name="BlueButton_up">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/button_up.png</normaloff>:/button_up.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="BlueButton_down">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/button_down.png</normaloff>:/button_down.png</iconset>
        </property>
       </widget>
       <widget class="QLabel" name="brightness_b">
        <property name="geometry">
         <rect>
          <x>35</x>
          <y>120</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
